### DISCUSSION TITLE
Kandinsky 2.2?

### DISCUSSION THREAD

**Discussion by seruva19** [seruva19/kubin#86] [General]
According to [this Telegram channel](https://t.me/complete_ai/131) (by one of the developers of Kandinsky), new "enhanced" version is on the way. 
Presumably, it is a more advanced finetune, but perhaps there will be something even more intriguing? 👀 And it also will be open-sourced.  

<details><summary>Teaser images</summary>

![photo_2023-05-25_21-40-15](https://github.com/seruva19/kubin/assets/26826215/254d19e7-326b-4212-8fd0-a7bbad304a17)
![photo_2023-05-25_21-40-20](https://github.com/seruva19/kubin/assets/26826215/8a007825-3417-49a1-a1e2-d2ad832ea6aa)
![photo_2023-05-25_11-49-26](https://github.com/seruva19/kubin/assets/26826215/13585436-a108-4105-9cd1-9a7af0783918)
![photo_2023-05-25_21-40-21](https://github.com/seruva19/kubin/assets/26826215/bebb3475-47c9-453c-84d4-60060973bd3a)
![photo_2023-05-25_21-40-22](https://github.com/seruva19/kubin/assets/26826215/27ef3826-5f2b-407f-bda6-83e943af0ccf)
![photo_2023-05-25_21-40-23](https://github.com/seruva19/kubin/assets/26826215/59fe3dfc-873c-45db-9f52-d03c7eeee362)
![photo_2023-05-25_21-40-24](https://github.com/seruva19/kubin/assets/26826215/68b1ae46-e90b-4074-b1a8-adf58a7e61ac)
![photo_2023-05-25_21-40-25 (2)](https://github.com/seruva19/kubin/assets/26826215/8ced56d4-1b5e-4d26-bb4c-e0aef1f57fd4)
![photo_2023-05-25_21-40-25](https://github.com/seruva19/kubin/assets/26826215/77346eb3-c882-405d-94c0-f3354c6e7465)

</details>

**Comment by Blucknote** [comment]
Probably yeah) Waitint release 👀

**Comment by user425846** [comment]
Any idea when this is going to release? I dont speak russian unfortunately

Evgeniy ***@***.***> schrieb am Do. 25. Mai 2023 um 22:12:

**Reply by Blucknote** [reply]
No clue. They are not share additional info. Probably soon

**Comment by adrianpuiu** [comment]
pff , i can not wait. look at that castle landscape ... it can do fingers to ... biting my nails already :))

**Comment by seruva19** [comment]
Nice to know that Kandinsky 2.2 will have support for ControlNet pipeline at launch 😀
https://github.com/huggingface/diffusers/pull/3936
https://huggingface.co/docs/diffusers/main/en/api/pipelines/kandinsky#diffusers.KandinskyV22ControlnetPipeline

**Reply by Blucknote** [reply]
They officially announced it! But no weights at this moment

**Reply by seruva19** [reply]
They are released now https://huggingface.co/kandinsky-community

**Reply by Blucknote** [reply]
Wait, community? what happened to [ai-forever](https://huggingface.co/ai-forever)?

**Reply by seruva19** [reply]
Good question) But I don't know :)

**Comment by user425846** [comment]
I just tried it out using the telegram bots. The results compared to 2.1
are very disappointing. They are much worse.

Evgeniy ***@***.***> schrieb am Mi. 12. Juli 2023 um 14:20:

**Reply by Blucknote** [reply]
I have to agree, just some prompts returning ok results

**Comment by seruva19** [comment]
Now THAT is really what I am waiting for ;)
https://github.com/huggingface/diffusers/pull/4051
